src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -456,4 +457,14 @@ public void receiveStartGame() {
             }
         }
     }
+
+    @Override
+    public void receiveOnBattleStart(AbstractRoom abstractRoom) {
+        if (ReapAndSowSaveData.reapAndSowDamage != 0) {
+            AbstractDungeon.actionManager.addToBottom(
+                    new DamageAllEnemiesAction(null, DamageInfo.createDamageMatrix(ReapAndSowSaveData.reapAndSowDamage, true),
```

Comment:
I think probably it would be better to use the `AbstractDungeon.player` as the damage source rather than null; it's still the player responsible for the damage, just on a delay

Reply:
makes perfect sense.